### PR TITLE
fusion results sample checks

### PR DESCRIPTION
**Pull request by kgaonkar6**
to check samples in recurrent fusion files are found in current histology file

### CODE REVIEW

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+root_dir is OpenPBTA-analysis. git cloned from https://github.com/AlexsLemonade/OpenPBTA-analysis#how-to-obtain-openpbta-data
+
+
+##### Steps
+
+1. PRINTS: dimentions of new and old datasets
+2. CHECKS: sample overlap between new and histology file
+3. CHECKS: 
+4. CHECKS:
```
Comment (jharenza) [inline OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: these are blank?

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+toAddpbtaFusionRecurrentlyFusedGenesBysample<-read_tsv(file.path(root_dir, params$toAddDataFolder,"pbta-fusion-recurrently-fused-genes-bysample.tsv"))
+
+# read histology file
+clinical<-file.path(root_dir, params$histology)
+histologyFile<-read_tsv(clinical,col_types = readr::cols(molecular_subtype = readr::col_character()))
+
+```
+
+#### v11 Stranded data
```
Comment (jharenza) [inline OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: should this be v13

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd (hunk 1 above)
Comment (kgaonkar6) [reply OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: we only had 1 check for this release : overlap of samples in rec_fused_genes files to the current histology so they are empty. 

In future releases if we add the breakpoints/domains etc. these checks will be used to find overlapping fusionNames in new and old dataset.

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd (hunk 2 above)
Comment (kgaonkar6) [reply OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: yeah should be v14 released files. I'll update that

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd (hunk 1 above)
Comment (jharenza) [reply OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: ok great

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd (hunk 2 above)
Comment (jharenza) [reply OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: ok

File: OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd [hunk 3]
```diff
@@ -0,0 +1,136 @@
+---
+title: "v13 QC RNAseq"
```
Comment (jharenza) [inline OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: also this should be v15

Comment (kgaonkar6) [reply OpenPBTA_v15_release_QC/QC_Fusion_v15_gene.Rmd]: ah yes I fixed that too

**Pull request merged**